params/config.go

```diff
@@ -476,7 +551,7 @@ func (c *ChainConfig) String() string {
 	default:
 		engine = "unknown"
 	}
-	return fmt.Sprintf("{ChainID: %v Homestead: %v DAO: %v DAOSupport: %v EIP150: %v EIP155: %v EIP158: %v Byzantium: %v Constantinople: %v Petersburg: %v Istanbul: %v, Muir Glacier: %v, Berlin: %v, London: %v, Arrow Glacier: %v, Engine: %v}",
+	return fmt.Sprintf("{ChainID: %v Homestead: %v DAO: %v DAOSupport: %v EIP150: %v EIP155: %v EIP158: %v Byzantium: %v Constantinople: %v Petersburg: %v Istanbul: %v , Muir Glacier: %v, Ramanujan: %v, Niels: %v, MirrorSync: %v, Berlin: %v, London: %v, Arrow Glacier: %v, Engine: %v}",
```

Comment:
Can you please print only Ethereum forks if BSC forks are all 0s, and print only BSC forks otherwise

Reply:
Let me know if the new code is what will work

Reply:
No, because for BSC test-nets it will still print Ethereum forks

Reply:
Check on consensus engine type - this should work

Reply:
Updated